Soccar [Game-Server]/src/nl/soccar/gameserver/model/GameServer.java

```diff
@@ -0,0 +1,83 @@
+package nl.soccar.gameserver.model;
+
+import nl.soccar.gameserver.controller.rmi.GameServerRmiController;
+import nl.soccar.gameserver.controller.socnet.GameServerSocnetController;
+import nl.soccar.gameserver.model.session.SessionController;
+import nl.soccar.gameserver.model.session.SessionWrapper;
+import nl.soccar.library.Session;
+import nl.soccar.socnet.connection.Connection;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author Lesley
+ */
+public final class GameServer {
+
+    private static final GameServer instance = new GameServer();
+
+    private final Map<Connection, PlayerWrapper> players = new HashMap<>();
+    private final SessionController sessionController = new SessionController();
+    private GameServerRmiController rmiController;
+    private GameServerSocnetController socnetController;
+
+    private GameServer() {
+    }
+
+    public static GameServer getInstance() {
+        return instance;
+    }
+
+    public void registerPlayer(PlayerWrapper player) {
+        registerPlayer(player.getConnection(), player);
+    }
+
+    public void registerPlayer(Connection connection, PlayerWrapper player) {
+        players.put(connection, player);
+    }
+
+    public void deregisterPlayer(PlayerWrapper player) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)